src/js/view/view.js

```diff
@@ -898,7 +936,15 @@ function View(_api, _model) {
 
             // Wrapper should inherit from parent unless floating.
             style(_playerElement, { backgroundImage: null }); // Reset to avoid flicker.
-            style(_wrapperElement, { width: null, height: null });
+            style(_wrapperElement, {
```

Comment:
Shouldn't these be reset when you stop dragging? Or what about snapping to place?

Reply:
What do you mean by snapping to place?

It is being called here because when we stop floating, `_wrapperElement` needs to go back to the original position. With `left` `right` `top` or `bottom` being set on the style, `_wrapperElement` doesn't do that properly.